src/Transports.Subscriptions.WebSockets/Extensions/GraphQLBuilderWebSocketsExtensions.cs

```diff
@@ -20,5 +20,18 @@ public static IGraphQLBuilder AddWebSockets(this IGraphQLBuilder builder)
 
             return builder;
         }
+
+        /// <summary>
+        /// Add required services for GraphQL web sockets
+        /// </summary>
+        public static DI.IGraphQLBuilder AddWebSockets(this DI.IGraphQLBuilder builder)
+        {
+            builder
+                .Register(typeof(IWebSocketConnectionFactory<>), typeof(WebSocketConnectionFactory<>), DI.ServiceLifetime.Transient)
+                .Register<IOperationMessageListener, LogMessagesListener>(DI.ServiceLifetime.Transient)
```

Comment:
I would not add `LogMessagesListener`. At least add bool or fluent argument:
```
public static DI.IGraphQLBuilder AddWebSockets(this DI.IGraphQLBuilder builder, Action<IWebSocketsBuilder> configure)
```

Then:
```
services.AddGraphQL(b => b.AddWebSockets(opt => opt.AddLogging(...)));
```

Reply:
Keep in mind that one of the disadvantages of the project I see unnecessary dependencies, namely `Microsoft.Extensions.Logging` and `Newtonsoft.Json`.

Reply:
For sure. Right now even when using System.Text.Json there is a dependency on Newtonsoft.Json. That’s one of the things I want to solve when embedding both serialization and deserialization into GraphQL.SystemTextJson and GraphQL.NewtonsoftJson in an abstract fashion.

Reply:
#680